assets/src/models/notificationData.ts

```diff
@@ -65,12 +66,27 @@ export const BridgeNotificationData = union([
   }),
 ])
 
+export const DetourNotificationData = type({
+  __struct__: literal(NotificationType.Detour),
+  detour_id: detourId,
+  headsign: string(),
+  route: string(),
+  direction: string(),
+  origin: string(),
+})
```

Comment:
Oh oops I didn't realize `status` wasn't on here.

Reply:
Will fix in followup PR

Reply:
I need to make this change here, because the update to `notification.ts` from the prior comment requires it

Reply:
Reverted! Instead, deleted the `deactivated` logic that was premature, so I didn't need to update `status`. Can happen in future PR